drivers/doc.txt

```diff
@@ -43,3 +43,10 @@
  * @ingroup     drivers
  * @brief       Drivers for storage devices
  */
+
+/**
+ * @defgroup    drivers_can CAN Drivers
+ * @ingroup     drivers
+ * @ingroup     can
```

Comment:
Yes, but I don't get, why we can't keep the name, just because CAN doesn't use the netdev API... After all, CAN stands for Controller Area *Network* ;)